dev/source-release.sh

```diff
@@ -109,7 +110,11 @@ fi
 # archive (identical hashes) using the scm tag
 echo "Creating tarball ${tarball} using commit $release_hash"
 tarball=$tag.tar.gz
-git archive $release_hash --worktree-attributes --prefix $tag/ -o $projectdir/$tarball
+git archive $release_hash --worktree-attributes --prefix $tag/ --add-file $projectdir/version.txt --add-file $projectdir/iceberg-build.properties -o $projectdir/$tarball
+
+# remove the uncommitted build files so they don't affect the current working copy
+rm $projectdir/version.txt
+rm $projectdir/iceberg-build.properties
```

Comment:
This wasn't needed before because the `version.txt` file was committed. When switching back to the master branch, it would be removed because it isn't present in the master branch.

Reply:
Will there be any commit specifically for the release or will it simply be the last commit that goes into the release branch? Previously the version.txt commit wound up being the source release commit.

To be clear, I am definitely in agreement on removing the `git commit` from this script. It has caused several people issues while testing releasing in the past or when the release script needed to be rerun for the same RC candidate.